.travis.yml

```diff
@@ -1,23 +1,41 @@
 language: cpp
+matrix:
```

Comment:
It looks like you removed support for `clang`. Could we please re-enable that?

Reply:
On Travis CI documentation they recommend that you use OSX images for `clang`. This reduces build time as well because it is preinstalled. If you still want me to revert that, that's ok!

Reply:
Ah, so xcode uses `clang` internally?

Reply:
Yes!!!